### DISCUSSION TITLE
Showing Related Jobs

### DISCUSSION THREAD

**Discussion by richyumul** [bullhorn/career-portal#405] [Ideas]
Say - I have the current codebase implemented, but the related jobs do not show up.  Is there a configuration item I need to tweak for them to display?

Beta Implementation (under testing)
https://jobs.provenrecruiting.com/test/#/jobs/14785

Old implementation
https://jobs.provenrecruiting.com/#/jobs/14785

**Comment by charlesabarnes** [comment]
Hello, we decided to not include related jobs in this new release as it hasn't seen much use. I will look into re implementing it in a future release

**Comment by moosewebdev** [comment]
Have the same request from my client for the return of the related jobs. Or having the link to view more jobs in this category

**Comment by ProcureAbility** [comment]
Same request, thx
